tools/app-toolkit/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+    "_note": "This file is only for IDE. For build use tsconfig.[dev/build].json (dev mode is less strict)",
+    "compilerOptions": {
+        "esModuleInterop": true,
+        "target": "es5",
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
```

Comment:
the same here depends when it will be merged but later we need it set to "lib": ["es2017", "dom"]